### DISCUSSION TITLE
Potentially driving an Arduino Nano off the Wand's Barrel-LED JST?

### DISCUSSION THREAD

**Discussion by ThePackDoctor** [gpstar81/GPStar-proton-pack#415] [Ideas]
How much voltage does the Barrel LED JST provide? I'm trying to figure out how I can drive an Arduino Nano for a custom vent light (uses a Neopixel jewel) and I'm wondering, could the Barrel LED JST be used to pull 5v while the wand is on? I don't think it's used in a Spengler wand, from the WAND_PCB page, as well as (I assume) being used for the lone LED board on the store page. 

I'm sorry for asking for a bunch of weird, esoteric features, but I've got some pretty neat stuff cooking up. 3 Red hat lights, two red Arcolectric lenses, a neopixel jewel in the vent, and a spirit pack that's gonna have a removable cyclotron and full internals. I'm super excited.

**Comment by nomakewan** [comment]
Closing as duplicate of #418 .
